### ISSUE TITLE
fix the document

### ISSUE THREAD

**Issue by hiepxanh** [angular-buch/angular-cli-ghpages#12]
![image](https://cloud.githubusercontent.com/assets/16970990/23655570/14eaf94e-0368-11e7-8e3e-ab9ca790ebcb.png)
angular CLI update to new version
you should fix 
`from:
ng build --prod --base-href "https://USERNAME.github.io/REPOSITORY/"
to :
ng build --prod --base-href "https://USERNAME.github.io/REPOSITORY/"`

this will make this won't render error Can't resolve './$$_gendir/app/app.module.ngfactory'

**Comment by JohannesHoppe** [comment]
hmmm... generally `--prod` and `--env-prod` should yield to exactly the same result?! :thinking: 

@fmalcher ?!

**Comment by JohannesHoppe** [comment]
This might be related to https://github.com/angular/angular-cli/issues/4294

**Comment by fmalcher** [comment]
`--prod` and `--env=prod` are actually **not** doing the same.
While `--env=prod` just selects the environment file, `--prod` selects the `production` target (which is the same like `--target=production`). This enables AOT compilation, minification and tree shaking.

In above screenshot you see that for `--env=prod` there are no hash values in the bundle file names. This indicates that the `development` target is enabled and thus, JIT compilation is taking place.

To sum up, changing the command to `--env=prod` does not solve the actual problem. It just bypasses the AOT issue by disabling AOT ;-) The problem is connected to the Angular CLI, not to `angular-cli-ghpages`.

**Comment by hiepxanh** [comment]
thanks for your explain
